### ISSUE TITLE
What about upload to the Play Store?

### ISSUE THREAD

**Issue by SEQUOiA87** [martinhellwig/DatapassWidget#7]
What do you say? I think many people could benefit from the widget & quick settings tile :)

But before doing this, I would like to add Firebase functionality (at least Analytics & Crash Reporting) to the app. I already played a bit with it and find it very useful. We can also discuss about Firebase in another Issue if you like to.

**Comment by mhettig** [comment]
I would appreciate this :) But I think there are a few small things / polishments to do before we can upload it.
